src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/amendhearingoutcome/AmendHearingOutcomeAboutToSubmitHandler.java

```diff
@@ -31,12 +37,62 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
             throw new IllegalStateException("Cannot handle callback");
         }
 
-        SscsCaseData sscsCaseData = callback.getCaseDetails().getCaseData();
+        CaseDetails<SscsCaseData> caseDetails = callback.getCaseDetails();
+        SscsCaseData sscsCaseData = caseDetails.getCaseData();
+
+        PreSubmitCallbackResponse<SscsCaseData> preSubmitCallbackResponse = new PreSubmitCallbackResponse<>(sscsCaseData);
+
+        List<String> hearingsSelected = new ArrayList<>();
 
         if (sscsCaseData.getHearingOutcomes().isEmpty()) {
             sscsCaseData.setHearingOutcomes(null);
+            return new PreSubmitCallbackResponse<>(sscsCaseData);
+        } else {
+            for (HearingOutcome selectedHearingOutcome :  sscsCaseData.getHearingOutcomes()) {
+                String selectedHearingId =
+                        selectedHearingOutcome.getValue().getCompletedHearings().getValue().getCode();
+                if (hearingsSelected == null) {
```

Comment:
is this ever going to be true given that we declare it as an empty arrayList a few lines above?
also, if it's null surely calling .add on it would trigger an NPE